src/Ot2Rec/params.py

```diff
@@ -204,15 +204,14 @@ def new_align_yaml(project_name: str):
         yaml.dump(align_yaml_dict, f, indent=4, sort_keys=False)
 
 
-def new_recon_yaml(project_name: str):
+def new_recon_yaml(args):
```

Comment:
Does `new_savurecon_yaml` also need to be updated to use argparse?

Reply:
Yes it does - I'll do that asap thanks